src/localisation/string_ids.h

```diff
@@ -2441,6 +2441,8 @@ enum {
 	STR_CHEAT_DISABLE_LITTERING_TIP = 5802,
 
 	STR_SHORTCUT_MUTE_SOUND = 5804,
+	
+	STR_SERVER_TIP_ADVERTISE = 5805,
```

Comment:
We always put `TIP` at the end. Please change the constant to
`STR_ADVERTISE_SERVER_TIP`